### ISSUE TITLE
uploaded results not found on website

### ISSUE THREAD

**Issue by paulwratt** [TheRemote/PiBenchmarks#9]
no errors were reported by the script, and I verified the length description to be less that others already posted

I did a test on the internal (Toshiba) sd-card which boots my **4Gb v1.1 RPi4**, listed as `6/25/20 10:22 am`:
https://storage.jamesachambers.com/benchmark/27985

I then tested a 128Gb Sandisk, which did not show up on the web page.

I then tested an WD Elements 1Tb 2.5" HDD, listed `6/25/20 10:46 am`:
https://storage.jamesachambers.com/benchmark/27988

I then retested the 128Gb sd-card, and another 64Gb Sandisk sd-card, both failed to upload.

The only thing they have in common is that they are are on card readers.

The 128Gb Sandisk Ultra is in a Lexar USB3 sd-card reader, connected via a Orico 2.5" SSD 3 port USB3 hub (without an SSD), connect to one of the USB3 ports
`Bus 002 Device 005: ID 05dc:b051 Lexar Media, Inc. microSD RDR UHS-I Card Reader [LRWM03U-7000]`

The 64Gb Sandisk is in a (used-to-be generic) multi-card reader, connected to one of the USB2 ports
`Bus 001 Device 004: ID 058f:6364 Alcor Micro Corp. AU6477 Card Reader Controller`

I did each test 2x, and did not use either `,` or `"` characters in the description, no errors were reported, but neither result was uploaded. the `lsusb` output listed above were both correctly identified by the script.

**Comment by paulwratt** [comment]
I notice today that the "Notes" field is empty, was that a fix for the db being down (cleared that field in the db)?

**Comment by TheRemote** [comment]
Hey paulwratt,

This was ages ago (sorry I didn't see it sooner).  It looks like I hadn't added some of your drives yet.

I'm working on the SD reader detection (I do have the data in there, it just needs to parse it for USB readers).  Your benchmark #27988 looks like I already fixed before I saw it as it is displaying live on the site now.

Thanks for the feedback, if you run up on anything else let me know and I will try to get to it much sooner next time!
